### PR TITLE
adapted code to ESH ThingHandler API refactoring

### PR DESCRIPTION
**Pull request by kaikreuzer**
See https://github.com/eclipse/smarthome/pull/2087

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java [hunk 1]
```diff
@@ -579,39 +578,6 @@ private void setDeviceConfiguration(Device device) {
...
-    @Override
-    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
-        logger.debug("Bridge Status updated to {} for device: {}", bridgeStatusInfo.getStatus().toString(),
-                getThing().getUID().toString());
-        getMaxCubeBridgeHandler();
-        if (bridgeStatusInfo.getStatus().equals(ThingStatus.ONLINE)) {
-            // No action
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
-            forceRefresh = true;
```
Comment (marcelrv) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: hi @kaikreuzer 
I assumed bridgeStatusChanged was not depreciated.  Is there a specific reason this was removed?

I guess now  `updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);` is no longer needed, but the `forceRefresh = true;` still is desired for the binding.

Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: sounds fair, I might have been too eager in removing code :-)

**Comment by marcelrv** [comment]
:-) I'll add it once more..

2016-11-27 19:24 GMT+01:00 Kai Kreuzer <notifications@github.com>:

File: addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java (hunk 1 above)
Comment (marcelrv) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: fixed the issue with https://github.com/openhab/openhab2-addons/pull/1495

**Pull request merged**